wrobot_auto_trial.py

```
@@ -10,13 +10,37 @@
+            windows_found = pywinauto.findwindows.find_windows(title="Trial version")
+            for hwnd in windows_found:
+                try:
+                    window_title = win32gui.GetWindowText(hwnd)
+                    print(f"Found Trial version popup: '{window_title}' (HWND: {hwnd})")
+                    # Send WM_CLOSE without stealing focus
+                    win32gui.PostMessage(hwnd, 0x0010, 0, 0)  # WM_CLOSE
+                    print("Dismissed Trial version popup (no focus steal)!")
+                except Exception as e:
+                    print(f"Error handling Trial version popup {hwnd}: {e}")
+        except Exception:
+            pass  # Silently continue if no windows found
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)